packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -195,11 +209,14 @@ export const run = (includeInternalScripts = false) => {
 
   program
     .command('plugin:ci-package')
-    .option('--signing-admin', 'Use the admin API endpoint for signing the manifest.', false)
+    .option('--signatureType <type>', 'Signature Type')
+    .option('--rootUrls <urls...>', 'Root URLs')
```

Comment:
It does not hurt to add here, but I don't think it can be used outside our internal CI now anyway

Reply:
Yeah, this is more to keep things consistent than anything else.